Makefile

```diff
@@ -44,7 +44,7 @@ start-test: ## Tests a previous build Docker image to see if starts
 	@docker images | grep -q "${NAME_BUILD_CONTAINER}" || \
 		(echo "[test] Docker image not found, running 'make build'" && make build)
 	@docker run -d --name="${NAME_TEST_CONTAINER}" \
-		-v /var/run/docker.sock:/var/run/docker.sock \
+		-v /var/run/docker.sock:/var/run/docker.sock --read-only --security-opt=no-new-privileges \
```

Comment:
i assume you've verified this all still works? since in the docs for this thing it even mentions that something you might need privileged mode

Reply:
yes, the testing was all done with these arguments